apps/api/src/app/portfolio/portfolio.service.ts

```diff
@@ -750,16 +764,28 @@ export class PortfolioService {
     return accounts;
   }
 
-  private getOrders(aUserId: string) {
+  private getOrders({
+    includeDrafts = false,
+    userId
+  }: {
+    includeDrafts?: boolean;
+    userId: string;
+  }) {
+    const where: Prisma.OrderWhereInput = { userId };
+
+    if (includeDrafts === false) {
+      where.isDraft = false;
+    }
+
     return this.orderService.orders({
+      where,
```

Comment:
In my opinion the entire query shouldn't be in this service

Reply:
Okay, I have moved `getOrders()` to the order service.